src/blocks/types.tsx

```diff
@@ -431,19 +432,6 @@ export const BlockTypes: { [key: string]: BlockType } = {
     createdBy: "jamesburet",
     fullscreenEdit: false,
   },
-  "Unknown": {
```

Comment:
undo delete here, we still need a fallback miniapp type!

Reply:
woops